util/types/src/core/tx_pool.rs

```diff
@@ -79,6 +79,13 @@ impl Reject {
             _ => false,
         }
     }
+
+    /// Returns true if the reject reason is low_fee_rate.
+    pub fn is_allowed_relay(&self) -> bool {
+        // Declared Wrong Cycles should be allowed to relay again
+        matches!(self, Reject::DeclaredWrongCycles(..))
+            || (!matches!(self, Reject::LowFeeRate(..)) && !self.is_malformed_tx())
+    }
```

Comment:
The method comment is "Returns true if the reject reason is low_fee_rate.", but there is a `!` in front of `matches!(self, Reject::LowFeeRate(..))`.

 so when `matches!(self, Reject::DeclaredWrongCycles(..))` is `false` and the reject reason is low_fee_rate, `fn is_allowed_replay` will return false.